README.md

```diff
@@ -4,6 +4,31 @@ Based on [Qiling](https://github.com/qilingframework/qiling) and [AFL++](https:/
 Written by Itai Liba ([@liba2k](https://twitter.com/liba2k)) and Assaf Carlsbad ([@assaf_carlsbad](https://twitter.com/assaf_carlsbad)).
 
 ## Usage
+
+### Using Docker enviourment
+
+1. Build image:\
```

Comment:
Need to `sudo apt install docker.io` before.
Also, `docker build` also seems to require sudo.

Reply:
I’m guessing you’ve installed docker from WSL, I’ve downloaded docker for windows installer… you can just start cmd and run docker build without sudo. 
I Don't think we should tell people how to install docker on their system, they can go to the site and read the installation instructions.